.travis.yml

```diff
@@ -26,13 +26,13 @@ matrix:
         - NUMPYSPEC="--pre --upgrade --timeout=60 -f $PRE_WHEELS numpy"
     - python: 3.6
       env:
-        - TESTMODE=full
+        - TESTMODE=slow
         - COVERAGE="--coverage --gcov"
         - NUMPYSPEC=numpy
     - python: 3.5
       env:
         - TESTMODE=fast
-        - COVERAGE=
+        - COVERAGE="--coverage --gcov"
```

Comment:
I wouldn't add coverage checking to the run that also runs refguide-check, since that's going to be slower.
Note also that codecov.yml needs to be edited to tolerate multiple reports.